src/main/java/com/xiaomi/infra/pegasus/rpc/async/ReplicaSession.java

```diff
@@ -207,6 +209,17 @@ public void operationComplete(ChannelFuture channelFuture) throws Exception {
     }
   }
 
+  private void startNegotiation(Channel activeChannel) {
```

Comment:
StartNegotiate

Reply:
I think startNegotiation is ok. negotiate is a verb.